src/sfdx-evergreen-auth/commands/cleanup.yml

```diff
@@ -0,0 +1,7 @@
+description: |
+  Removes unencrypted credentials.
+steps:
+  - run:
+      name: Clean-up sfdx evergreen auth
+      command: |
+        rm .netrc || true
```

Comment:
Could this affect other uses of netrc? Maybe if this is meant to run at the end of the job it doesn't matter much.